Makefile

```diff
@@ -29,8 +29,8 @@ lint:
 build-all:
 	$(call eachmod,go build -tags balsam -o /dev/null ./...)
 
-docker:
-	docker build -t ipfs/testground .
+docker-sidecar:
+	docker build -t iptestground/sidecar:edge .
```

Comment:
The semantics I use are:
`edge` - latest master build
`latest` - latest stable release (currently v0.4.0)
`v0.4.0`, `v0.3.0`, etc.

Reply:
Also if someone picks the source for a given release (v0.4.0 or v0.3.0, etc.), this will always be building latest master. I know we are not pulling images from testground, but maybe we should re-consider that... other how am I going to run release v0.4.0 after we release v0.5.0 - what/who is going to pull the sidecar image with the correct version for me?